src/main/java/org/umssdiplo/automationv01/core/managepage/reports/AvailableItemsReport.java

```diff
@@ -0,0 +1,35 @@
+
+package org.umssdiplo.automationv01.core.managepage.reports;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.umssdiplo.automationv01.core.managepage.BasePage;
+import org.umssdiplo.automationv01.core.utils.CommonEvents;
+
+public class AvailableItemsReport extends BasePage {
+
+    @FindBy(id = "first-column-header")
+    private WebElement firstColumnTitle;
```

Comment:
these columns have some name probably we can use the name column intead of first second